test/lib/ansible_test/_internal/sanity/__init__.py

```diff
@@ -8,6 +8,8 @@
 import re
 import collections
 
+from distutils.version import LooseVersion
```

Comment:
Use `str_to_version` instead of `LooseVersion`:

https://github.com/ansible/ansible/blob/d7618712fd13804de07dacf06c64e7ac34c395fc/test/lib/ansible_test/_internal/util.py#L861-L863

We don't need more complex version handling for Python versions in this case.

Reply:
Didn't knew that one existed. Using it now. Thanks!